sympy/core/tests/test_symbol.py

```diff
@@ -366,3 +367,27 @@ def test__uniquely_named_symbol_and__symbol():
     assert F('x1', Symbol('x1'),
         modify=lambda i: i + '_').name == 'x1_'
     assert _symbol(x, _x) == x
+
+
+def test_disambiguate():
+    x, y, y_1, _x, _x_1, x_1, x_2 = symbols('x y y_1 _x _x_1 x_1 x_2')
```

Comment:
I guess `_x_1` must be removed if it is not used. If you say so I will remove it.